api/shape/view.js

```diff
@@ -18,8 +18,8 @@ export const baseView =
     wireframe = false,
     prepareView = (x) => x,
     inline,
-    width = 1024,
```

Comment:
Does this make views smaller by default?

Reply:
Yes, but I think they were rendering smaller anyhow -- when I fixed the sizes they got very large. :)